docs/releases/5.1.txt

```diff
@@ -116,8 +116,8 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   720,000 to 870,000.
 
-* The default ``parallelism`` of the ``ScryptPasswordHasher`` is 
-  increased from 1 to 5, to follow OWASP recommendations.
+* The default ``parallelism`` of the ``ScryptPasswordHasher`` is increased from
```

Comment:
There was a trailing space on this line. In addition to removing it, I made the line longer since it seemed inconsistent.